packages/components/package.json

```diff
@@ -48,6 +52,7 @@
         "postcss-import": "^16.0.1",
         "prettier": "^3.2.5",
         "react": "^18.2.0",
+        "react-dom": "^18.2.0",
```

Comment:
Please add a `peerDependencies` too (seems like I forgot it 😄 )

Reply:
done